### DISCUSSION TITLE
White mist

### DISCUSSION THREAD

**Discussion by PeanuthOToole** [vladmandic/sdnext#3424] [Q&A]
Ever since a recent update a white mist covers all my image generations. I reset all settings to default, tried different checkpoints, different samplers. Note: "mist" was not in the prompt ;)
![00006-epicrealism_naturalSinRC1VAE-car](https://github.com/user-attachments/assets/6acd7654-c715-491b-b0c2-8514071ae541)
![00007-Realistic_Vision_V5 1-car](https://github.com/user-attachments/assets/5eec8bf2-26c7-4fcd-ba82-a51ef8595c98)

**Comment by vladmandic** [comment]
please create an issue and document everything - this is most likely related to some vae setting, but i don't even know which backend or model type you're using.

**Reply by PeanuthOToole** [reply]
Issue sorted itself out magically.
